recipe/meta.yaml

```diff
@@ -1,14 +1,20 @@
+{% set name = "django-extensions" %}
+{% set version = "1.6.7" %}
+{% set md5 = "62dd2a173ad29f319b84f1b89571c060" %}
+{% set download = "https://pypi.python.org/packages/7f/da/1245fe47d1a2ca8d2d03b3cdbc886e64c94a8c2d3b6f99e3464d507a7c29/django-extensions-1.6.7.tar.gz" %}
```

Comment:
@kwilcox can you use:

`https://pypi.io/packages/source/d/django-extensions/django-extensions-1.6.1.tar.gz` instead of that monstrosity PyPI is forcing upon us :wink: